collaboration/communication.md

```diff
@@ -9,39 +9,64 @@
 
 # Communication
 
-______________________________________________________________________
-
-## Communication Schedule
-
-| Day | How | The topic of discussion | | --- | :-: | ----------------------- |
-| | | |
-
-## Communication Channels
-
-how often will we get in touch on each channel, and what we will discuss there:
-
-- **Issues**:
-- **Pull Requests**:
-- **Slack/Discord**:
-- **Video Calls**:
-
-______________________________________________________________________
+_## Communication Schedule_
+...
+
+| Day       | How                 | The Topic of Discussion     |
+|-----------|---------------------|-----------------------------|
+| Monday    | Google Meet/Slack   | Goal Setting               |
+| Tuesday   | Google Meet/Slack   | Code Review                |
+| Wednesday | Google Meet/Slack   | Code Review and Help       |
+| Thursday  | Google Meet/Slack   | Progress Update            |
+| Friday    | Google Meet/Slack   | Code Review and Help       |
+| Saturday  | Google Meet/Slack   | Code Review and Help       |
+| Sunday    | Google Meet/Slack   | Progress Update            |
+
+_## Communication Channels_
+...
+
+We will always be in touch to discuss various things:
+
+- **Issues**: For bugs, progress checking, and other related tasks.
+- **Pull Requests**: For code reviews and Markdown file updates.
+- **Slack**: For daily updates, notifications, and reporting any code
+- errors or help needed.
+- **Video Calls**: For group discussions, progress updates, tutoring, and meetings.
+- **WhatsApp**: For informal hangouts and check-ins.
 
 ## Availability
 
-### Availability for calling/messaging
-
-| Day | Monday | Tuesday | Wednesday | Thursday | Friday | Saturday | Sunday | |
------- | :----: | :-----: | :-------: | :------: | :----: | :------: | :----: |
-| _name_ | | | | | | | |
-
-### How many hours everyone has per day
-
-- name: _5h_;
-- name: _6h_;
-- name: _5h_;
-- name: _4h_;
-- name: _3h_;
+_### Availability for Calling_
+...
+
+| Member| Monday | Tuesday | Wednesday | Thursday | Friday  | Saturday | Sunday |
+|-------------|------------|------------|------------|------------|------------|------------|------------|
+| Dadi    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm |
+| Member 2    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 3    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 4    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 5    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 6    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 7    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+| Member 8    | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm | 5 - 8pm|
+
+_### How Many Hours Everyone Has Per Day on Average_
+...
+
+- **Dadi**: 8 hours
+- **Rama**: # 4 hours
+- **Member 3**: # of hours
```

Comment:
Ahmed: 4 hours

Reply:
the same thing, you have to add yourself